cmd/qat_plugin/README.md

```diff
@@ -68,6 +68,7 @@ $ make  intel-qat-plugin
 ### Deploy QAT device plugin as a DaemonSet:
 ```
 $ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
+kubectl create -f deployments/qat_plugin/qat_plugin_default_configmap.yaml
```

Comment:
We can have a single command here which can be used for deploying both the configmap and daemonset of the device plugin i.e. kubectl create -f deployments/qat_plugin

Reply:
I thought about this but then it means the user needs to modify the deployment yaml when making config changes. With this, the expected flow is: 1) prepare the configuration, 2) do the deployment with an unmodified qat_plugin.yaml.